src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/seed/cas1/ApprovedPremisesRoomsSeedFromXLSXJob.kt

```diff
@@ -26,11 +26,15 @@ class ApprovedPremisesRoomsSeedFromXLSXJob(
   private val approvedPremisesRepository: ApprovedPremisesRepository,
   private val roomRepository: RoomRepository,
   private val bedRepository: BedRepository,
-  private val siteSurvey: SiteSurvey,
+  private val characteristicRepository: CharacteristicRepository,
 ) : ExcelSeedJob {
   private val log = LoggerFactory.getLogger(this::class.java)
+  private val questionCriteriaMapping = QuestionCriteriaMapping(characteristicRepository)
+  private lateinit var qCode: String
```

Comment:
this shouldn't be a class instance variable to avoid concurrency issues

Reply:
Will revisit this when work is done to address [APS-1757](https://dsdmoj.atlassian.net/browse/APS-1757).